packages/wrangler/src/init.ts

```diff
@@ -462,9 +526,14 @@ export async function initHandler(args: ArgumentsCamelCase<InitArgs>) {
 				await mkdir(path.join(creationDirectory, "./src"), {
 					recursive: true,
 				});
-
+				const serviceMetaData = await fetchResult<ServiceMetaDataRes>(
+					`/accounts/${accountId}/workers/services/${fromDashScriptName}`
+				);
+				const defaultEnvironment =
+					serviceMetaData["default_environment"]["environment"];
+				// I want the default environment, assuming it's the most up to date code.
```

Comment:
If we are supporting environments, then perhaps we should (in the near future) add a way to specify the environment when setting the dash worker name?

Reply:
Or perhaps if there is more than one environment, we could offer a selection prompt with a list of available environments to choose from?

Reply:
I think that can be added in the future.